src/App.js

```diff
@@ -1,7 +1,70 @@
 import Background from "./assets/Background.svg";
 import Logo from "./assets/spacecenter1.svg";
+import { useNavigate } from "react-router-dom";
 
 function App() {
+  const LOGIN_API_URL = "http://localhost:3000/auth/login";
+
+  const CREATE_USER_URL = "http://localhost:3000/users";
+
+  const navigate = useNavigate();
+
+  async function user(mail, pass) {
+    const response = await fetch(CREATE_USER_URL, {
+      method: "POST",
+      body: JSON.stringify({
+        email: mail,
+        password: pass,
+      }),
+      headers: {
+        "Content-type": "application/json; charset=UTF-8",
+      },
+    });
+    const json = await response.json();
+    return json;
+  }
+
+  async function login(mail, pass) {
+    const response = await fetch(LOGIN_API_URL, {
+      method: "POST",
+      body: JSON.stringify({
+        email: mail,
+        password: pass,
+      }),
+      headers: {
+        "Content-type": "application/json; charset=UTF-8",
+      },
+    });
+    const json = await response.json();
+    return json;
+  }
+
+  const handleSubmit = async () => {
+    const mail = document.getElementById("email").value;
+    const pass = document.getElementById("password").value;
+    const confirmEmail = document.getElementById("confirmEmail").value;
+    const confirmPassword = document.getElementById("confirmPassword").value;
+
+    if (
+      mail === confirmEmail &&
+      pass === confirmPassword &&
+      mail &&
+      confirmEmail &&
+      pass &&
+      confirmPassword
```

Comment:
There's a small improvement we could make here cc: @MFJess 

[JS has short-circuit evaluation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND#short-circuit_evaluation), meaning it stops evaluating when it finds a `false` value in an AND comparison

As example, in `if (false && ...)` it doesn't matter what else is after the `&&` when there's already a `false` value

In here, you are first comparing if the email is equal to the confirmation (same thing for password) and only after you are checking if even you had an email value to begin with. If we submitted the email fields empty, we would be doing comparisons with empty values before we realize the email is empty. As alternative, we could first check if the email value exists and only after we make the check that it matches the confirmation

In this particular case, it's not that really relevant but it's something to keep in mind when you start working with objects and accessing properties inside, for example, `if (user && user.name == "davide")`. If you had this order reversed, you'd get an error when the `user` is not defined because you'd be trying to access the `name` property of `null`